packages/commonwealth/client/scripts/state/api/trpc/subscription/useCommunityAlertsQuery.ts

```diff
@@ -1,5 +1,15 @@
 import { trpc } from 'utils/trpcClient';
 
-export function useCommunityAlertsQuery() {
-  return trpc.subscription.getCommunityAlerts.useQuery({});
+export function useCommunityAlertsQuery({
+  enabled = true,
+}: {
+  enabled?: boolean;
+}) {
+  return trpc.subscription.getCommunityAlerts.useQuery(
+    {},
+    {
+      enabled,
+      staleTime: Infinity,
```

Comment:
Community alerts change but only on user input and since all community alert mutations invalidate cache on success, there is no reason to refresh this cache periodically.

Reply:
thanks